kivymd/uix/filemanager.py

```diff
@@ -653,8 +658,8 @@ def get_content(self):
                                 pass
                         else:
                             if (
-                                not self.show_hidden_files
-                                and content.startswith(".")
+                                    not self.show_hidden_files
+                                    and content.startswith(".")
```

Comment:
This is not according to the `PEP8` standard.

Reply:
Hmm, PyCharm did it, not me)